pyproject.toml

```diff
@@ -249,6 +250,7 @@ exclude = [
     "node_modules",
     "venv",
     "tests/assets",
+    "src/otx/algo/**/mmdeploy/",
```

Comment:
It's minor, but can you put this down to 258 lines?

Reply:
but It's necessary until we use mmdeploy.